### PR TITLE
Add storage to lockup contract

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
https://github.com/near/core-contracts/blob/a4c0bf31ac4a5468c1e1839c661b26678ed8b62a/lockup/src/lib.rs#L33
image

### CODE REVIEW

**Comment by Megha-Dev-19** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@Megha-Dev-19 Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Weekly streak is on the road, smart strategy! Secure your streak with another PR!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/171&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/171&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Megha-Dev-19)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Megha-Dev-19)

For maintainers:
- Score pull requests that participate in the Race of Sloths and receive a reward
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

File: playwright-tests/tests/stake-delegation/stake-delegation.spec.js
Comment (petersalomonsen) [inline playwright-tests/tests/stake-delegation/stake-delegation.spec.js:1006]: It would be better to show the actual args here and use a function for converting to base64. Makes the code more readable

File: playwright-tests/tests/stake-delegation/stake-delegation.spec.js
Comment (petersalomonsen) [inline playwright-tests/tests/stake-delegation/stake-delegation.spec.js:1304]: Why is this test removed?

Comment (Megha-Dev-19) [reply playwright-tests/tests/stake-delegation/stake-delegation.spec.js:1304]: actually we don't allow user to add a stake/unstake request, for more than the available/staked amount, so I removed the checks from the 'Approve' action, since it won't happen, and other dao frontend doesn't have UI to add staking request.

File: playwright-tests/tests/stake-delegation/stake-delegation.spec.js
Comment (Megha-Dev-19) [reply playwright-tests/tests/stake-delegation/stake-delegation.spec.js]: yeah good suggestion, I wasn't able to reproduce the same base64 w/o Big, but installed it.

**Comment by petersalomonsen** [comment]
@race-of-sloths score 5

**Review by petersalomonsen — APPROVED** [review]

**Pull request merged**
